README.md

```diff
@@ -48,7 +48,7 @@ For detailed installation instructions and requirements, see the [Installation G
 
 ### Quick Start
 
-#### 1a. Start an OpenAI Compatible Server (vLLM)
+#### 1. Start an OpenAI Compatible Server (vLLM)
```

Comment:
Perhaps offer a link to how to install vllm?

Reply:
Good idea, can link out to those docs on vLLM website.